src/main/java/com/rackspace/ceres/app/model/TsdbQueryRequest.java

```diff
@@ -0,0 +1,13 @@
+package com.rackspace.ceres.app.model;
+
+import java.util.List;
+import java.util.Map;
+
+import lombok.Data;
+
+@Data
+public class TsdbQueryRequest {
+  String metric;
+  String downsample;
+  List<Map<String, String>> filters;
```

Comment:
I think instead of creating this as Map of String we can create new class as `Filters` with the fields `type` which can again be an Enum having `literal_or` right now, `tagk` and `filter` as defined in http://opentsdb.net/docs/build/html/api_http/query/index.html

Reply:
@harageth What's your say on this one?

Reply:
done